addons/im_livechat/static/src/js/im_livechat_channel_form_controller.js

```diff
@@ -1,7 +1,6 @@
-odoo.define('im_livechat.ImLivechatChannelFormController', function (require) {
-'use strict';
+/** @odoo-module alias=im_livechat.ImLivechatChannelFormController **/
 
-const FormController = require('web.FormController');
+import FormController from 'web.FormController';
```

Comment:
Yes, we can use the non-alias name in imports. But should we do it?  Will it be very strange and not consistent because these web modules are always aliased?

Reply:
Ok, fair enough